docs/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "docs:dev": "pnpm prebuild && vitepress dev",
+    "docs:dev": "pnpm prebuild && pnpm prepare && vitepress dev",
```

Comment:
Most probably `pnpm prepare` should be added to the other two commands too: `docs:build` and `docs: preview`, otherwise they will fail with the same error. What do you think about adding `postinstall` script and run the `prepare` command there?

Reply:
Yeah good idea, will add them there as well.
I'm not in favor of adding it to `postinstall`, this will run every time when installing packages. This will run when working with packages.